core/src/main/java/fr/sncf/osrd/api/stdcm/STDCMEndpoint.java

```diff
@@ -108,14 +109,16 @@ public Response act(Request req) throws InvalidRollingStock, InvalidSchedule {
                     request.gridMarginAfterSTDCM,
                     request.gridMarginBeforeSTDCM
             );
+            DriverBehaviour driverBehaviour = new DriverBehaviour();
```

Comment:
I'd prefer if we don't change the STDCM behavior here. It won't work as it is (the offsets should extend from one edge to the other), and I don't think it's really a feature we want by default.